test/b2gedmntuples_cfg.py

```diff
@@ -100,6 +102,7 @@
 process.load('Configuration.StandardSequences.MagneticField_38T_cff')
 process.load('Configuration.StandardSequences.Services_cff')
 process.load('Configuration.StandardSequences.FrontierConditions_GlobalTag_condDBv2_cff')
+process.load("RecoEgamma/PhotonIdentification/PhotonIDValueMapProducer_cfi")
```

Comment:
Hi, Savvas,

For 3 I meant did you update the Readme with the prescription needed, or is this up to date?

Cheers,
Sal